opensrs/opensrsapi.py

```diff
@@ -651,3 +651,21 @@ def _make_registrant_verification_call(self, domain_name, operation):
             object='domain',
             attributes={'domain': domain_name}
         ).get_data()
+
+    def enable_auto_renew(self, cookie, domain):
+        self._set_auto_renew_status(cookie, domain, True)
+
+    def disable_auto_renewal(self, cookie, domain):
```

Comment:
Since this client can operate other objects (except for domains), I think it makes sense to add `domain` to the methods' names.  E.g.:

`enable_domain_auto_renewal()`, `disable_domain_auto_renewal()`  and `_set_domain_auto_renewal_status()`